components/script/dom/webidls/WebGLRenderingContext.webidl

```diff
@@ -654,6 +654,9 @@ interface WebGLRenderingContextBase
     [Throws]
     void texImage2D(GLenum target, GLint level, GLenum internalformat,
                     GLenum format, GLenum type, TexImageSource? source); // May throw DOMException
+    [Throws, Pref="dom.webgl.dom_to_texture.enabled"]
```

Comment:
This does not actually throw any exceptions, so we can remove the annotation.

Reply:
It still throws the ```let pipeline_id = source.pipeline_id().or_or(Error::InvalidState)?;```exception